python/ray/_private/usage/usage_constants.py

```diff
@@ -6,7 +6,18 @@
 # The name of a json file where usage stats will be written.
 USAGE_STATS_FILE = "usage_stats.json"
 
-USAGE_STATS_HEADS_UP_MESSAGE = (
-    "Usage stats collection will be enabled by default in the next release. "
-    "See https://github.com/ray-project/ray/issues/20857 for more details."
+USAGE_STATS_ENABLED_MESSAGE = (
+    "Usage stats collection is enabled. To disable this, add `--disable-usage-stats` to the command that starts the cluster, or run the following command:"
+    " `ray disable-usage-stats`. "
+    "See https://github.com/ray-project/ray/issues/20857 for more details.\n"
 )
+
+USAGE_STATS_DISABLED_MESSAGE = "Usage stats collection is disabled."
+
+USAGE_STATS_ENABLED_BY_DEFAULT_MESSAGE = (
+    "Usage stats collection is enabled by default without user confirmation since it is running non-interactively. To disable this, add `--disable-usage-stats` to the command that starts the cluster, or run the following command:"
+    " `ray disable-usage-stats`. "
+    "See https://github.com/ray-project/ray/issues/20857 for more details.\n"
+)
+
+USAGE_STATS_CONFIRMATION_MESSAGE = "Enable usage stats collection. See https://github.com/ray-project/ray/issues/20857 for more details. It will be enabled if there is no confirmation within 10s."
```

Comment:
```suggestion
USAGE_STATS_CONFIRMATION_MESSAGE = "Enable usage stats collection? See https://github.com/ray-project/ray/issues/20857 for more details. It will be enabled if there is no confirmation within 10s."
```

Reply:
Sorry I forgot to push, could you check the latest PR?